packages/markdown-docx/src/ToOOXMLVisitor/rules.js

```diff
@@ -283,7 +283,8 @@ const OPTIONAL_RULE = (title, tag, value, type) => {
 const VANISH_PROPERTY_RULE = () => '<w:vanish/>';
 
 /**
- * Inserts Baskerville Old Face font family OOXML tag.
+ * Inserts Baskerville Old Face font family OOXML tag to distinguish nodes of whenNone/whenFalse
```

Comment:
Try to put yourself in someone who is not that familiar with this code.

They try to interpret this function and they found out that it has something to do with optional or condition nodes. They look at the CiceroMark properties and see that in fact, `whenSome` and `whenNone` exist. However, they would definitely question "what are `whenTrue` and `whenFalse` for?" Given then context, they would deduce that `whenTrue` and `whenFalse` are also attributes in the CiceroMark which is false!

In fact this is true wherever you have metioned the above terms so you need to rectify that. Probably a better wording would be:

```
Inserts a different fontface so that the hidden contents can be distinguished. There can be two types of hidden content -
1) when `hasSome` is true, the content inside `whenNone` is hidden.
2) when `hasSome` is false, the content inside `whenSome` is hidden.
```

I know this takes time and it may not be commensurate for the time you spend thinking over it. But I can promise you that you only will forget the same code you have written 4 months later given you don't work on this project for that long.

Reply:
@algomaster99 Actually, this is not quite the right description. the font family is for `else` condition always. It does not matter whether they are hidden or visible.

Reply:
If they are visible, then we do not want to change the font.

Reply:
@algomaster99 
Does this sound right?
```md
 Inserts a different font family so that the conditional/optional content can be distinguished. They can be of following types:-
 1) `whenNone` nodes for optional(to be distinguished from `whenSome`)
 2) `whenFalse` nodes for conditional(to be distinguished from `whenTrue`)
```

Reply:
Okay, so the font face has been introduced to handle conditional as well? I thought you would make a separate PR for that.

Reply:
@algomaster99 
It will automatically deal with that when I make the PR for conditional. `Optional` and `Conditional` are identically same. Just some node keys changing.

Reply:
@algomaster99 
I will update the JSDOC for conditional in that PR. Currenly, I will only make it for optional. Does it sounds fine?

Reply:
Sounds good. You could have applied the same logic for the variables too but that's okay. We can merge this one.

Anyway, just try to focus on one feature at a time. It causes confusion otherwise.